unusedtest/disk.go

```diff
@@ -30,4 +31,5 @@ func (d Disk) CreatedAt() time.Time      { return d.createdAt }
 func (d Disk) Meta() unused.Meta         { return d.meta }
 func (d Disk) LastUsedAt() time.Time     { return d.createdAt.Add(1 * time.Minute) }
 func (d Disk) SizeGB() int               { return d.size }
+func (d Disk) SizeBytes() int            { return d.size * int(math.Pow(1000, 3)) }
```

Comment:
Ditto nit: Providers charge per GiB, which is 1024 ^3

Reply:
I'm going for bytes so we guarantee consistency on the metrics we get, since AWS provides the volume sizes in GiB, while GCP uses GB 🙄 Surprisingly enough, Azure is the only one that provides both values out of the box.